.github/workflows/style.yml

```diff
@@ -37,17 +42,21 @@ jobs:
       run: ./script/clang-format --check
 
     - name: CMake Format
+      if: ${{ always() }}
```

Comment:
What does this do?

Reply:
Runs even if the thing fails. Perhaps it should be a separate PR witth these ci changes?

Reply:
Or, perhaps you questioned the sanity of doing it to the first check? If so, I have no answer

Reply:
Could you separate it into its own commit?